libsplinter/src/biome/credentials/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Defines a basic API to register and authenticate a SplinterUser using a username and a password.
+//! Not recommended for use in production.
```

Comment:
I would use the phrase "This feature is experimental, and not necessarily production-ready."

Reply:
I thought that `biome-credentials` was never meant to be used in production (even when it is no longer is experimental), as most companies have their own ways to create and authenticate users. `biome-credentials` was more of a dev tool.